src/get-started/install/windows_quick.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Windows install
+description: How to install on Windows.
+short-title: Windows
+next:
+  title: Set up an editor
+  path: /get-started/editor
+---
+
+{% assign os = 'windows' -%}
+
+## System requirements
+
+To install and run Flutter,
+your development environment must meet these minimum requirements:
+
+- **Operating Systems**: Windows 10 or later (64-bit), x86-64 based.
+- **Disk Space**: 1.64 GB (does not include disk space for IDE/tools).
+- **Tools**: Flutter depends on these tools being available in your environment.
+  - [Windows PowerShell 5.0][] or newer (this is pre-installed with Windows 10)
+  - [Git for Windows][] 2.x, with the
+    **Use Git from the Windows Command Prompt** option.
+
+     If Git for Windows is already installed,
+     make sure you can run `git` commands from the
+     command prompt or PowerShell.
+
+{% include_relative _get-sdk-win_v2.md %}
+
+{% include_relative _android-setup.md %}
```

Comment:
Make sure you leave out any of these `% include`s you don't want, right now the full Android instructions are being included on this page.

```suggestion
```